REGION_FORK_RECOMMENDATIONS.md

```diff
@@ -0,0 +1,193 @@
+# Region Specific Fork Recommendations
+
+This document describes recommendations for keeping forks of [covidgreen/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api) clean while also allowing for region specific changes. For example, the New York State fork [project-vagabond/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api) contains CICD workflows specific to NYS which should not be contributed back to Covid Green. These recommendations adhere to the project's [Contributing Guidelines](CONTRIBUTING.md).
+
+## Goals and When to Use
+
+These recommendations exist for the situations when a Public Health Authority is basing their Covid Tracing system on Covid Green but potentially making region specific changes. There are 4 overall goals to these recommendations
+  
+1. All changes, unless explicitly region specific, will be contributed back to [covidgreen/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api).
+1. Region specific forks will stay up-to-date with [covidgreen/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api).
+1. Region specific changes will never be contributed back to [covidgreen/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api).
```

Comment:
the 1 and 3 points are essentially the same. Perhaps they can be combined?

Reply:
I think it's important to explicitly state what must be and what cannot be contributed back to Covid Green. I've re-ordered and rephrased the bullets to hopefully clarify that.